tests/e2e_test.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import asyncio
+import os
+import time
+from pathlib import Path
+from typing import cast
+
+import numpy as np
+import pennylane as qml
+import requests
+from compute_api_client import ApiClient, Configuration, Member, MembersApi, PageMember
+
+from pennylane_quantuminspire2.api.pagination import PageReader
+from pennylane_quantuminspire2.api.settings import ApiSettings, AuthSettings, TokenInfo
+from pennylane_quantuminspire2.qi_device import QI2Device
+
+
+async def _fetch_team_member_id(host: str, access_token: str) -> int:
+    config = Configuration(host=host, access_token=access_token)
+    async with ApiClient(config) as client:
+        page_reader = PageReader[PageMember, Member]()
+        members_api = MembersApi(client)
+        pagination_handler = page_reader.get_single
+        members_handler = members_api.read_members_members_get
+        member = await pagination_handler(members_handler)
+        if member is None:
+            raise RuntimeError("Member does not exist")
+        return cast(int, member.id)
+
+
+def _get_auth_tokens() -> None:
```

Comment:
This probably causes you problems to authenticate. I see the failures in the the actions.
Look at how it is done in the platform:

https://gitlab.com/qutech-sd/quantum-inspire-2/platform/-/blame/master/ci/scripts/get_oauth_tokens.py?ref_type=heads#L10